lib/src/main/java/com/auth0/android/lock/PasswordlessLock.java

```diff
@@ -110,33 +109,34 @@ public static Builder newBuilder(@NonNull LockCallback callback) {
     /**
      * Builds a new intent to launch LockActivity with the previously configured options
      *
-     * @param activity a valid Activity context
+     * @param context a valid Context
      * @return the intent to which the user has to call startActivity or startActivityForResult
      */
     @SuppressWarnings("unused")
-    public Intent newIntent(Activity activity) {
-        Intent lockIntent = new Intent(activity, PasswordlessLockActivity.class);
+    public Intent newIntent(Context context) {
+        Intent lockIntent = new Intent(context, PasswordlessLockActivity.class);
         lockIntent.putExtra(Constants.OPTIONS_EXTRA, options);
+        lockIntent.addFlags(Intent.FLAG_ACTIVITY_NEW_TASK);
         return lockIntent;
     }
 
     /**
      * Should be called on the Activity holding the Lock instance's OnDestroy method, as it
```

Comment:
"This method ensures proper Lock's lifecycle handling. Must be called from the class holding the Lock instance whenever you're done using it. i.e. in the Activity's onDestroy method."